code/modules/uplink/uplink_item.dm

```diff
@@ -455,22 +455,22 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	include_modes = list(/datum/game_mode/nuclear)
 
 /datum/uplink_item/ammo/shotgun/bag
-	name = "12g Ammo Duffelbag"
-	desc = "A duffelbag filled with enough 12g ammo to supply an entire team, at a discounted price."
+	name = "12g Ammo Duffel bag"
```

Comment:
Will switch it over. Thought that spelling was consistent elsewhere and that duffle was the outlier.